services/app/src/reducers/Search.tsx

```diff
@@ -4,10 +4,12 @@ import {SearchChangeParamsAction, SearchResponseAction} from '../actions/ActionT
 import {getStorageString, setStorageKeyValue} from '../LocalStorage';
 import {SearchState} from './StateTypes';
 
+const LANGUAGE = 'language';
+
 export const initialSearch: SearchState = {
   results: null, // null = need to search; [] = no results
   params: {
-    language: getStorageString('language', 'English') as LanguageType,
```

Comment:
What parses `English`? should we be concerned about matching case here and elsewhere? Can we pull this default from a constant somewhere else?

Reply:
The casting itself doesn't actually guarantee it'll be correct, true. That being said, 'English' is used 21 times in the codebase, and I'm sure the problem is in several other places, so I've filed a code cleanliness issue to address it more holistically: https://github.com/ExpeditionRPG/expedition/issues/502